Helpers/get_results_from_criteria_and_data.py

```diff
@@ -0,0 +1,78 @@
+from datetime import date, timedelta
+from typing import List
+
+from graphql_objects import DatesCheck
+
+
+class GetResultsFromSearchCriteriaAndData():
+
+    @staticmethod
+    def get_data_check_results(validated_search_criteria, id_houses_dict: dict) -> List[DatesCheck]:
```

Comment:
Use `typing.Dict` to explicitly define what is inside `id_houses_dict`.

Reply:
Also, id_to_houses_dict is a far better name, without the "to", it's not clear if you have a mapping of some unknown key to an object called "id houses", or if it's a mapping of ids to houses, etc.